tests/appsec/iast/source/test_cookie_value.py

```diff
@@ -14,9 +14,10 @@ class TestCookieValue(BaseSourceTest):
     requests_kwargs = [{"method": "GET", "cookies": {"table": "user"}}]
     source_type = "http.request.cookie.value"
     source_name = "table"
-    source_value = "user"
+    source_value = None if context.library.library == "nodejs" else "user"
```

Comment:
Why is source_value `None` for nodejs?

Reply:
Because `source_value` is redacted

Reply:
and why is redacted only in nodejs and not in java, for example?

Reply:
ummm, I checked them again and the values are not redacted. But the tests were ok. I don't understand what is going on here :S